{"name": "test_loginsalesforce_validcredentials[test_list_item0]", "status": "broken", "statusDetails": {"message": "TypeError: 'SalesforceUtils' object is not callable", "trace": "self = <testcases.login_test.TestLogin object at 0x000001C312529BD0>, test_list_item = {'password': 'Pra245pte#', 'username': 'pgbitbucket706@agentforce.com'}\n\n    @pytest.mark.smoke\n    @pytest.mark.parametrize(\"test_list_item\", BaseTest.get_data_from_json(Login_test_data_path,\"valid_login\"))\n    def test_loginsalesforce_validcredentials(self, test_list_item):\n        loginpage = LoginPage(self.driver)\n        homepage = loginpage.logintosalesforce(test_list_item[\"username\"],test_list_item[\"password\"])\n        #homepage.verify_page_title()\n>       homepage.verify_contact_details_by_lastname()\n\ntestcases\\login_test.py:20: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\homePage.py:21: in verify_contact_details_by_lastname\n    contact_details = sf_utils.get_contact_detail_by_lastname('Young',10)\n                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nutility\\salesforce_data_utils.py:21: in get_contact_detail_by_lastname\n    sf = self.verify_salesforce_authentication()\n         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nauthenticate_salesforce = <utility.salesforce_data_utils.SalesforceUtils object at 0x000001C31256D6A0>\n\n    def verify_salesforce_authentication(authenticate_salesforce):\n>       sf_auth = authenticate_salesforce()\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^\nE       TypeError: 'SalesforceUtils' object is not callable\n\nutility\\salesforce_data_utils.py:15: TypeError"}, "parameters": [{"name": "test_list_item", "value": "{'username': 'pgbitbucket706@agentforce.com', 'password': 'Pra245pte#'}"}], "start": 1750346527316, "stop": 1750346539610, "uuid": "c245de06-522d-4cef-8e09-188894293bad", "historyId": "4fa26c5d8a8f3f216283ca707f04bde8", "testCaseId": "edd327156eda62a83bae9df650959eeb", "fullName": "testcases.login_test.TestLogin#test_loginsalesforce_validcredentials", "labels": [{"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-QPHGE425"}, {"name": "thread", "value": "21600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.login_test"}]}